### PR TITLE
OCM-5440: OCP-65645: Mp reconciliation feature TC

### PR DESCRIPTION
**Pull request by amalykhi**
Ran 1 of 50 Specs in 16.752 seconds
SUCCESS! -- 1 Passed | 0 Failed | 1 Pending | 48 Skipped
PASS

Ginkgo ran 1 suite in 17.715053201s
Test Suite Passed

### CODE REVIEW

File: tests/e2e/machine_pool_test.go [hunk 1]
```diff
@@ -614,6 +614,41 @@ var _ = Describe("TF Test", func() {
...
+
+				By("Delete machinepool by OCM API")
+				cms.DeleteMachinePool(ci.RHCSConnection, clusterID, mpName)
+				_, err = cms.RetrieveClusterMachinePool(ci.RHCSConnection, clusterID, mpName)
+				Expect(err).To(HaveOccurred())
+				Expect(err.Error()).Should(ContainSubstring("Machine pool with id '%s' not found", mpName))
+
+				By("ReApply the machinepool manifest")
+				_, err = mpService.Apply(MachinePoolArgs, false)
+				Expect(err).ToNot(HaveOccurred())
```
Comment (shellymiron) [inline tests/e2e/machine_pool_test.go:648]: maybe worth adding another `By("Verify the parameters of the created machinepool")` ?
just to check the machine pool creation?

**Comment by bardielle** [comment]
/override ci/prow/rosa-sts-advanced-optional-presubmit 
it should be fixed by yuwan's PR (mentioned [here](https://redhat-internal.slack.com/archives/C063JMV5SEP/p1704273812714219?thread_ts=1704222832.976629&cid=C063JMV5SEP)). If no PR by tomorrow, I'll remind him

**Pull request merged**